### ISSUE TITLE
Admin shutdown button doesn't offer any options

### ISSUE THREAD

**Issue by manics** [jupyterhub/jupyterhub#3880]
### Bug description
Reported in https://discourse.jupyter.org/t/what-does-the-shutdown-hub-button-really-do/14023

The Shutdown button in the admin interface causes an instant(?) shutdown, it does not offer options to keep the proxy or singleuser servers running.

#### Expected behaviour

It looks like there's code to show a dialog:
https://github.com/jupyterhub/jupyterhub/blob/6be3160d7487bb362df7095f24454ef2814951fc/share/jupyterhub/static/js/admin.js#L279-L295
Though it's not getting called

#### Actual behaviour

JupyterHub is shutdown immediately, no options are offered. The proxy is stopped, but it looks like managed services aren't (this might be a separate issue?).

### How to reproduce

https://github.com/jupyterhub/jupyterhub/tree/main/examples/service-whoami
with the addition of
```python
c.JupyterHub.load_roles = [
    {
        'name': 'admin',
        'users': ['demo'],
    },
]
```
to `jupyterhub_config.py`

Login as `demo` and go to http://localhost:8000/hub/admin
Click Shutdown

**Comment by minrk** [comment]
Indeed, this is in the transition from the old admin page to the new react one. I believe `admin.js` is unused and can be removed.
